src/cmd/ksh93/sh/main.c

```diff
@@ -135,11 +134,6 @@ int sh_main(int ac, char *av[], Shinit_f userinit) {
     bool rshflag; /* set for restricted shell */
     char *command;
 
-#ifdef AST_SERIAL_RESTART
-    // Restart all ast_*() intercepted syscalls on EINTR.
-    astserial(AST_SERIAL_RESTART, AST_SERIAL_always);
-#endif
-    free(malloc(64 * 1024));
```

Comment:
Strange! Not sure what this was intended for.

Reply:
I'm hoping it was something like an attempt to "prime the pump" so that subsequent Vmalloc calls would be a trifle faster. Regardless, it's obviously silly.